src/main/java/com/practicesoftwaretesting/cart/model/CartItem.java

```diff
@@ -0,0 +1,17 @@
+package com.practicesoftwaretesting.cart.model;
+
+import com.practicesoftwaretesting.product.model.Product;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CartItem {
+    private String id;
+    private int quantity;
+    private Double discountPercentage;
```

Comment:
please decide, do you want to use primitives for int and double, or you want to use wrapper classes